reference/api/export.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Export
+description: "Migrate between instances with the `/export` route"
+---
+
+import { RouteHighlighter } from '/snippets/route_highlighter.mdx'
+
+import CodeSamplesExperimentalDeleteLogsStream1 from '/snippets/samples/code_samples_experimental_delete_logs_stream_1.mdx';
+
+Use the `/export` route to transfer data from your origin instace to a remote target instance. This is particularly useful when migrating from your local development environment to a Meilisearch Cloud instance.
+
+## Migrate database between instances
+
+<RouteHighlighter method="POST" path="/export" />
+
+Migrate data from the origin instance to a target instance.
+
+### Body
+
+| Name                | Type   | Default value | Description                                                                                               |
+| ------------------- | ------ | ------------- | --------------------------------------------------------------------------------------------------------- |
+| **`url`** *         | String | `null`        | The target instance's URL address. Required                                                               |
+| **`apiKey`** *      | String | `null`        | An API key with full admin access to the target instance                                                  |
+| **`payloadSize`** * | String | "50 MiB"      | A string specifying the payload size in a human-readable format                                           |
+| **`indexes`** *     | Object | `null`        | A set of patterns matching the indexes you want to export. Defaults to all indexes in the origin instance |
+
+#### `url`
+
+A string pointing to a remote Meilisearch instance, including its port if necessary.
+
+This field is required.
+
+#### `apiKey`
+
+A security key with full admin permissions to a secured Meilisearch instance.
```

Comment:
It requires the right to create indexes, update settings and upload documents, at least for now.